html/semantics/embedded-content/the-video-element/video_timeupdate_on_seek.html

```diff
@@ -0,0 +1,45 @@
+<!DOCTYPE HTML>
+<html>
+<head>
+    <title>HTML5 Media Elements: timeupdate is emitted after a seek before the data is received.</title>
+    <meta content="text/html; charset=UTF-8" http-equiv="Content-Type">
+    <link rel="author" title="Alicia Boya García" href="mailto:aboya@igalia.com"/>
+    <script src="/resources/testharness.js"></script>
+    <script src="/resources/testharnessreport.js"></script>
+</head>
+<body>
+<script>
+    const seekTime = 60 * 4;
+
+    function testTimeupdateOnSeek(mediaType) {
+        async_test(function (test) {
+            const video = document.createElement("video");
+            video.src = `timeout_on_seek.py?extension=${mediaType}`;
+            video.controls = true;
+            document.body.appendChild(video);
+
+            video.addEventListener("canplay", test.step_func(videoLoaded), {once: true});
+
+            function videoLoaded() {
+                video.addEventListener("timeupdate", test.step_func(onTimeUpdate));
+                video.play();
+                video.currentTime = seekTime;
+            }
+
+            function onTimeUpdate() {
+                if (Math.abs(video.currentTime - seekTime) <= 1) {
+                    test.done();
+                    document.body.removeChild(video);
```

Comment:
What's the intent behind removing the video from the DOM? If the goal is to reset the player, why not `video.src = "";`?

Reply:
That it does not remain in the DOM after the test has run.

Reply:
If the test fails (doesn't reach `test.done()`), then the video also won't be removed. The `test.add_cleanup()` function can be used to remove the video, right after it's inserted.

Reply:
But when the test fails often you are interested in what state it was at that point so you can debug the issue.

Reply:
Hmm yes, good point.